docs/productrequirementdocuments/product-requirements-document-template.md

```diff
@@ -0,0 +1,47 @@
+# Product Requirements Document Template
+
+
+## Objective
+
+Explain the customer problem we are solving and how it relates to our vision, goals, and initiatives. This will establish the high-level purpose of what we want to accomplish and for whom. This also keeps the broader team focused on building features that delight our customers — resulting in us delivering a MLP
```

Comment:
What does MLP stand for?

Reply:
Minimum Lovable Product but im going to remove that bit.